tests/attr/test_feature_ablation.py

```diff
@@ -109,6 +151,36 @@ def test_multi_input_ablation_with_mask(self):
             ablations_per_eval=(1, 2, 3),
         )
 
+    def test_multi_input_scalar_ablation_with_mask(self):
+        net = BasicModel_MultiLayer_MultiInput()
+        inp1 = torch.tensor([[23.0, 100.0, 0.0], [20.0, 50.0, 30.0]])
+        inp2 = torch.tensor([[20.0, 50.0, 30.0], [0.0, 100.0, 0.0]])
+        inp3 = torch.tensor([[0.0, 100.0, 10.0], [2.0, 10.0, 3.0]])
+        mask1 = torch.tensor([[1, 1, 1]])
+        mask2 = torch.tensor([[0, 1, 2]])
+        mask3 = torch.tensor([[0, 1, 2]])
+        expected = (
+            [[1784.0, 1784.0, 1784.0]],
+            [[160.0, 1200.0, 240.0]],
+            [[16.0, 880.0, 104.0]],
+        )
+        # testing int, float, 0d and 1d tensor output
+        for func in [
+            lambda *inp: torch.sum(net(*inp)).item(),
+            lambda *inp: torch.sum(net(*inp)),
+            lambda *inp: torch.sum(net(*inp)).reshape(1),
+            lambda *inp: int(torch.sum(net(*inp)).item()),
```

Comment:
nit: perhaps we can split this into four tests ? This will reduce test-wise execution time and will help to narrow down the failing case.